src/main/resources/application.yaml

```diff
@@ -37,7 +37,7 @@ spring:
             response-type: code
             response-mode: form_post
             prompt: login
-            issuer-uri: ${ACTIVE_DIRECTORY_B2C_BASE_URI}/${AAD_B2C_TENANT_ID:00000000-0000-0000-0000-000000000000}/v2.0/
+            issuer-uri: ${ACTIVE_DIRECTORY_B2C_BASE_URI:https://hmctsstgextid.b2clogin.com}/${AAD_B2C_TENANT_ID:00000000-0000-0000-0000-000000000000}/v2.0/
```

Comment:
Is this default url mean't to be exposed?

Reply:
This is already in the readme document I have defaulted here for simplicity